src/artwork.c

```diff
@@ -200,6 +202,12 @@ static struct artwork_source artwork_item_source[] =
       .data_kinds = (1 << DATA_KIND_SPOTIFY),
       .cache = ON_SUCCESS,
     },
+    {
+      .name = "Spotify web api",
+      .handler = source_item_spotifywebapi_get,
+      .data_kinds = (1 << DATA_KIND_SPOTIFY),
+      .cache = ON_SUCCESS,
```

Comment:
I would add ON_FAILURE. It is not included for libspotify because there is a timer on getting the artwork, and because there is no real overhead from asking libspotify several times.